receiver/elasticsearchreceiver/scraper.go

```diff
@@ -56,7 +57,10 @@ var (
 	}
 )
 
+var es7_10 *version.Version
+
 func init() {
+	es7_10, _ = version.NewVersion("7.10")
```

Comment:
```suggestion
var es7_10 *version.Version = version.NewVersion("7.10")

func init() {
```

Reply:
![Screen Shot 2022-09-12 at 5 28 24 PM](https://user-images.githubusercontent.com/13648435/189761876-a41223d1-15aa-4016-970a-07d2958203bd.png)

It expects 2 values,  `(*version.Version, error)`

Reply:
Ok, use an anonymous func then:
```go
var es7_10 = func() *version.Version {
    v, _ := version.NewVersion("7.10")
    return v
}()
```